Assets/MirageXR/Tests/NewUI/ProfileView.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ProfileView : PopupBase
+{
+    [SerializeField] private MoodleServersView _moodlePrefab;
+    [SerializeField] private Button _selectServer;
```

Comment:
let's add the prefix `_btn` to the button links

Reply:
done